markets/perps-market/subgraph/subgraph.base-goerli.yaml

```diff
@@ -60,6 +60,8 @@ dataSources:
             uint128,uint256,int256,int256,int128,int128,uint256,uint256,uint256,uint256,indexed
             bytes32,address)
           handler: handleOrderSettled
+        - event: OrderCancelled(indexed uint128, indexed uint128, uint256, uint256, int128, uint256, indexed bytes32, address)
```

Comment:
subgraphs cannot be updated in the same PR because they rely on deployed and published changes after code changes are already merged and released.